### DISCUSSION TITLE
Documentation and examples for @typespec/xml are not useful.

### DISCUSSION THREAD

**Discussion by todurnatesh** [microsoft/typespec#3943] [General]
Hello everyone,
        I am looking to have xml output with namespace / attributes for a particularly simple model. Was not able to implement these as the basic examples in https://typespec.io/docs/libraries/xml/guide dont work and the guide seems to be incomplete.
     
  Here is my basic setup.
  
 tspconfig.yml
 
```
emit:
  - "@typespec/openapi3"
 ```
 
 package.json
 
 ```
 {
  "name": "typespec-demo",
  "version": "0.1.0",
  "type": "module",
  "dependencies": {
    "@typespec/compiler": "0.58.0",
    "@typespec/http": "0.58.0",
    "@typespec/openapi3": "0.58.0",
    "@typespec/rest": "0.58.0",
    "@typespec/xml": "0.58.0"
  }
}
```

main.tsp

```
import "@typespec/http";
import "@typespec/xml";
import "@typespec/openapi3";

using TypeSpec.Xml;

@ns("https://example.com/ns1", "ns1")
model Foo {
  @ns("https://example.com/ns1", "ns1")
  bar1: string;

  @ns("https://example.com/ns2", "ns2")
  bar2: string;
}
```

### Test 
running `tsp compile .` in root directory.

### Current output
```
openapi: 3.0.0
info:
  title: (title)
  version: 0.0.0
tags: []
paths: {}
components:
  schemas:
    Foo:
      type: object
      required:
        - bar1
        - bar2
      properties:
        bar1:
          type: string
        bar2:
          type: string
```

### Expected output - 
 must have xml and namespace related attributes.
 
 Any help is appreciated.
 
 Thanks

**Comment by timotheeguerin** [comment]
The xml library currently is not used by the openapi emitter. It was added to start looking at some of the azure services like blob storage and make sure we are able to represent everything with them.

Realized we never created a issue for that, just create one https://github.com/microsoft/typespec/issues/3945 feel free to thumbs up to track user requests
